wlutil/wlutil.py

```diff
@@ -426,6 +420,8 @@ def initialize():
     for d in initramfs_disk_dirs:
         if not (ctx['initramfs-dir'] / 'disk' / d).exists():
             (ctx['initramfs-dir'] / 'disk' / d).mkdir(parents=True)
+        if not (ctx['initramfs-dir'] / 'nodisk' / d).exists():
```

Comment:
What is this `nodisk` that I see now?

Reply:
Marshal creates separate `disk` and `nodisk` sysroots for disk/initramfs builds. Previously, the requested busybox was only being copied into the `disk` side, which meant that initramfs builds did not use the correct busybox.

This fixes that.